### DISCUSSION TITLE
Support for Wii Guitar Hero Controller

### DISCUSSION THREAD

**Discussion by Nave478** [ndeadly/MissionControl#202] [Ideas]
Hello. Would be possible to add support for the Wii Guitar Hero attachment? Also thank you for developing MissionControl!

**Comment by ndeadly** [comment]
Sure, it's possible, I just never bothered because I didn't think there was any game that could use it in any meaningful way. Did you have something in mind?

**Reply by Nave478** [reply]
I don't really have any specific games I would use it for. I think it would be fun to try out with some rhythm games on the switch.

**Reply by ndeadly** [reply]
> some rhythm games on the switch

Do they exist? I can add support for it, but the controls need to be mapped to the layout of a Pro Controller/Joy-con in order for the switch to pick them up. I have no idea what to use for a default mapping.
